.github/workflows/test.yaml

```diff
@@ -0,0 +1,35 @@
+name: test
+
+on: [pull_request]
+
+jobs:
+  build:
```

Comment:
Consider a better name here too, maybe even just "pytest" since that's the main point of this job?

Reply:
I changed the filename and top level name, but adding pytest here feels redundant